### PR TITLE
Fix for xdr-blacklist-files and xdr-whitelist-files commands

### PR DESCRIPTION
**Pull request by bziser**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-19255)

## Description
Fix for xdr-blacklist-files and xdr-whitelist-files commands.
The yml showed they are deprecated, but by mistake they were removed from the py file.

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/4_9_10.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Updated the Docker image to: *demisto/python3:3.10.9.40422*.
+
+- Fixed an issue in ***xdr-blacklist-files*** and ***xdr-whitelist-files***.
```
Comment (dansterenson) [inline Packs/CortexXDR/ReleaseNotes/4_9_10.md:6]: same

File: Packs/Core/ReleaseNotes/1_3_21.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Investigation & Response
+- Fixed an issue in ***xdr-blacklist-files*** and ***xdr-whitelist-files***.
```
Comment (dansterenson) [inline Packs/Core/ReleaseNotes/1_3_21.md:4]: Please give more details. maybe `Fixed an issue where the ***xdr-blacklist-files*** and ***xdr-whitelist-files*** deprecated commands were mistakenly removed.`

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 3]
```diff
@@ -2077,7 +2077,7 @@ def allowlist_files_command(client, args):
     if detailed_response:
         return CommandResults(
             readable_output=tableToMarkdown('Allowlist Files', res),
-            outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.blocklist',
+            outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.allowlist',
```
Comment (dansterenson) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:2080]: Isn't is suppose to be like this?
```suggestion
            outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.{args.get("prefix", "allowlist")}',
```

**Review by dansterenson — CHANGES_REQUESTED** [review]

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py (hunk 3 above)
Comment (bziser) [reply Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:2080]: `detailed_response` is only in whitelist and blocklist

**Review by dansterenson — APPROVED** [review]

**Pull request merged**